### ISSUE TITLE
Indentation when unique child on same line than parent

### ISSUE THREAD

**Issue by simhayoz** [simhayoz/eval-code-quality#27]
```java
/* ... */
    public static boolean test() { return true; }
/* ... */
```
should not be an error, of _indentation 36 instead of 4_, because the child is on the same line as the parent

**Comment by simhayoz** [comment]
Was actually fixed on #14, but a similar bug exists on braces check

**Comment by simhayoz** [comment]
Fixed in #39 for the braces check part
